src/leveldbwrapper.h

```diff
@@ -168,6 +199,44 @@ class CLevelDBWrapper
     {
         return pdb->NewIterator(iteroptions);
     }
+
+    /**
```

Comment:
I'd put the implementation of these functions in leveldbwrapper.cpp instead of the header file (unless there's a reason I'm overlooking)